### PR TITLE
[Task Automation] Adds stopwords support for nltk

### PR DESCRIPTION
**Pull request by ssachdev-states**
**Intent**: Adding stopwords support for nltk. 

image

### CODE REVIEW

File: underwriter/Dockerfile [hunk 1]
```diff
@@ -263,6 +263,7 @@ RUN groupadd --gid 3434 circleci \
 
 # Install pipenv
 RUN sudo pip install pipenv
+RUN sudo pip install nltk && python3 -m nltk.downloader stopwords
```
[the remark this answers is not in the document]
Comment (ssachdev-states) [reply underwriter/Dockerfile:266]: Just installing nltk isn't enough. We have to download the words as an add-on. There are many libraries of words. The words we need are the stop words only.

[the remark this answers is not in the document]
Comment (zlongcheng) [reply underwriter/Dockerfile:266]: I am not sure we want to install specific Pyhthon package in shared Docker image. We handle Python dependency in individual projects' Pipfile in the mono repo, don't we? The point to use Python Virtual env is that individual project can have its own dependency.

[the remark this answers is not in the document]
Comment (karlhepler) [reply underwriter/Dockerfile:266]: I think I'm inclined to agree with @zlongcheng, unless you have a specific reason to include this with all builds.

[the remark this answers is not in the document]
Comment (ssachdev-states) [reply underwriter/Dockerfile:266]: So, you're suggesting to add this at an app level? @zlongcheng

[the remark this answers is not in the document]
Comment (zlongcheng) [reply underwriter/Dockerfile:266]: Yes

**Pull request closed without merging**